docs/content/en/docs/concepts/architecture/components/metrics-operator/_index.md

```diff
@@ -33,22 +46,40 @@ style Metrics-Adapter fill:#d8e6f4,stroke:#fff,stroke-width:px,color:#006bb8
-which can be used to gain insight into the behavior and performance of applications running on a Kubenetes cluster.
+The **Metrics adapter** exposes custom metrics from an application
+to external monitoring and alerting tools.
+The adapter exposes custom metrics on a specific endpoint
+where external monitoring and alerting tools can scrape them.
+It is an important component of the metrics operator
+as it allows for the collection and exposure of custom metrics,
+which can be used to gain insight into the behavior and performance
+of applications running on a Kubenetes cluster.
+
+The **Metrics controller** fetches metrics from an SLI provider.
+The controller reconciles a [`KeptnMetric`](../../../../yaml-crd-ref/metric.md)
+CR and updates its status with the metric value
+provided by the selected metric provider.
+Each `KeptnMetric` is identified by `name`
+and is associated with an instance of an observability platform
+that is defined in a
+[KeptnMetricsProvider](../../../../yaml-crd-ref/metricsprovider.md)
+CR.
 
-**Metrics controller** is used to fetch metrics from a SLI provider.
-The controller reconciles a [`KeptnMetric`](../../../../yaml-crd-ref/metric.md) CR and
-updates its status with the metric value provided by the selected SLI provider.
 The steps in which the controller fetches metrics are given below:
 
-* It first fetches the `KeptnMetric` object to reconcile.
-* If the object is not found, it returns and lets Kubernetes handle deleting all associated resources.
-* If the object is found, the code checks that if the metric has been updated within the configured
-interval which is defined in the `Spec.FetchIntervalSeconds`.
-If not, then it skips reconciling and requeues the request for later.
-* If the metric should be reconciled, it fetches the provider defined in the `Spec.Provider.Name` field.
-* If the provider is not found, it returns and requeues the request for later.
-* If the provider is found, it loads the provider and evaluates the query defined in the `Spec.Query` field.
-* If the evaluation is succesful, it stores the fetched value in the status of the `KeptnMetric` object.
+1. Fetch the `KeptnMetric` object to be reconciled.
+   * If the object is not found,
+     it returns and lets Kubernetes handle deleting all associated resources.
+1. If the object is found, the code checks
+   whether the metric has been updated within the interval
+   that is defined in the `spec.fetchintervalseconds` field.
+   * If not, it skips reconciling and requeues the request for later.
+1. If the metric should be reconciled,
+   it fetches the provider defined in the `spec.provider.name` field.
+   * If the provider is not found,
+     it returns and requeues the request for later.
+1. If the provider is found,
+   it loads the provider and evaluates the query
+   defined in the `spec.query` field.
+1. If the evaluation is successful,
+   it stores the fetched value in the status of the `KeptnMetric` object.
```

Comment:
if the evaluation fails the error and reason will also be there

Reply:
properties:
              errMsg:
                description: ErrMsg represents the error details when the query could
                  not be evaluated
                type: string
              lastUpdated:
                description: LastUpdated represents the time when the status data
                  was last updated
                format: date-time
                type: string
              rawValue:
                description: RawValue represents the resulting value in raw format
                format: byte
                type: string

Reply:
I added this information but it's not clear to me how to view this information.  I slogged through the autogenerated CRD/API info and I find a bit of "MetricStatus" stuff for autoscaling under ListMeta but nothing that really lines up here.  And how does one inspect this metric status data?  Does one use the standard way of inspecting a resource?

Reply:
https://github.com/keptn/lifecycle-toolkit/pull/1807 adds KeptnMetricStatus resource and other status resources to the API ref and that clarifies everything!  I am updating the arch piece to reference that resource.